regtests/ww3_tp2.14/input_oasicm/namcouple

```diff
@@ -29,7 +29,7 @@
 # debug and time statistic information written by OASIS3-MCT for each model
 # and processor (see user manual for more details)
 #
- 1
+ 1 0
```

Comment:
The problem with the winds was that the regtest is using the ww3_shel.inp file, and this file was not properly set. Instead of having an empty list of sent coupled fields, one needs to specify 'UNSET'.

Reply:
OK - results look ok to me now.
I am running the full regression test matrix.

@MatthewMasarik-NOAA and @JessicaMeixner-NOAA : We will need to add a new `density.nc` file to the `ww3_from_ftp.${ww3ver}.tar.gz` tar file for regtest `ww3_tp2.14/OASACM6`. Should we send this to you? It is only a small file (21Kb)

Reply:
Hi @ukmo-ccbunney, yes that would be great.  Please send it to me and we will update it on the FTP site.

Reply:
> Hi @ukmo-ccbunney, yes that would be great. Please send it to me and we will update it on the FTP site.
Thanks @MatthewMasarik-NOAA  - I've emailed you the new file.

Reply:
Received.  Thank you @ukmo-ccbunney.